File: leetcode/485.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        currCnt = 0
        maxCnt = 0
        for i in range(len(nums)):
            if nums[i] == 0:
                maxCnt = max(maxCnt, currCnt)
                currCnt = 0
            else:
                currCnt += 1

        return max(maxCnt, currCnt)

File: leetcode/485.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int currCnt = 0;
        int maxCnt = 0;
        for (int i = 0; i < nums.length; i++) {
            if (nums[i] == 0) {
                maxCnt = Math.max(currCnt, maxCnt);
                currCnt = 0;
            }
            else {
                currCnt++;
            }
        }
        maxCnt = Math.max(currCnt, maxCnt);
        return maxCnt;
    }
}